vortexasdk/endpoints/voyages_search_enriched_result.py

```diff
@@ -21,9 +22,11 @@ class VoyagesSearchEnrichedFlattenedResult(Result):
 
     def to_list(self) -> List[VoyageEnrichedItem]:
         # noinspection PyTypeChecker
-        raise Exception(f"to_list method is not supported for search results in the flattened format (i.e. when the `columns` API param is provided). Please use to_df() instead.")
+        raise Exception(
+            f"to_list method is not supported for search results in the flattened format (i.e. when the `columns` API param is provided). Please use to_df() instead."
+        )
 
-    def to_df(self) -> pd.DataFrame:
+    def to_df(self, columns: Optional[List[str]] = None) -> pd.DataFrame:
```

Comment:
Different from usual?

Reply:
The inferred type was specifically annotated here, but I've removed it for consistency.